### ISSUE TITLE
Can I use MKL in the BLAS option ?

### ISSUE THREAD

**Issue by gaoyanfei** [HiFiLES/HiFiLES-solver#139]
I saw these words in the code

`#if` defined `_MKL_BLAS`
`#include` `"mkl.h"`
`#include` `"mkl_spblas.h"`
`#endif`

But in the configure_run.sh, only ATLAS, STANDARD, ACCLERATE are accepted. I tried to add MKL include and lib, but it fails.

**Comment by mlopez14** [comment]
Yes, you should be able to use the MKL libraries using a custom Makefile.

I suggest you take a look at the sample Makefiles we have and modify one of
them to include the details of your MKL installation.

**Comment by venky187** [comment]
Hi all,

I tried compiling with MKL library and I am facing trouble running the code. I maybe wrong but I think the code is not compatible with MKL. Is anyone else facing this issue? I have attached log files from both ATLAS library and MKL library for the flatplate testcase. Please let know if anything needs to be done.

[log_ATLAS.txt](https://github.com/HiFiLES/HiFiLES-solver/files/872175/log_ATLAS.txt)
[log_MKL.txt](https://github.com/HiFiLES/HiFiLES-solver/files/872176/log_MKL.txt)
